src/components/specific/projects/project-card/ProjectCard.vue

```diff
@@ -16,10 +16,11 @@
             @open-viewer="goToModelViewer"
             @open-menu="openMenu"
           />
-          <div class="project-card__left-stripe"></div>
-          <div class="project-card__status-badge">
-            {{ $t("ProjectStatusBadge.active") }}
-          </div>
+          <div
+            class="project-card__left-stripe"
```

Comment:
why are you keeping this line ? You have a status for every conditions, then line 21 will always over-ride that one, isn't it ?

Reply:
No, line 21 won't override the styles associated with this class.
Here is the explanation:
This class (`.project-card__left-stripe`) hold the styles for positioning + with/height of the stripe.
Then the class with modifier (`.project-card__left-stripe--{status}`) set a specific color according to project status.

Here are the corresponding rules in `ProjectCard.scss`: 
```scss
.project-card__left-stripe {
   position: absolute;
   z-index: 1;
   width: 4px;
   height: $card-height;

   &--new {
      background-color: var(--color-primary);
   }

   &--active {
      background-color: var(--color-success);
   }

   &--inactive {
      background-color: var(--color-high);
   }

   &--paused {
      background-color: var(--color-tertiary-dark);
   }
}
```

If I remove this line (n° 20), then I should write the following (which contain many code repetition.. ) :
```scss
.project-card__left-stripe {
   &--new {
      position: absolute;
      z-index: 1;
      width: 4px;
      height: $card-height;
      background-color: var(--color-primary);
   }

   &--active {
      position: absolute;
      z-index: 1;
      width: 4px;
      height: $card-height;
      background-color: var(--color-success);
   }

   &--inactive {
      position: absolute;
      z-index: 1;
      width: 4px;
      height: $card-height;
      background-color: var(--color-high);
   }

   &--paused {
      position: absolute;
      z-index: 1;
      width: 4px;
      height: $card-height;
      background-color: var(--color-tertiary-dark);
   }
}
```

This is why I keep this line.

Reply:
**Note:** The naming convention for the classes comes from [the BEM methodology](http://getbem.com/).
See also: https://css-tricks.com/bem-101/

Reply:
thank you for this explanation, it's quite clear ! I will dive into the BEM methodology then

Paul

Reply:
You're welcome. ;-)